.settings/org.eclipse.jdt.core.prefs

```diff
@@ -304,7 +304,7 @@ org.eclipse.jdt.core.formatter.parentheses_positions_in_method_invocation=common
 org.eclipse.jdt.core.formatter.parentheses_positions_in_switch_statement=common_lines
 org.eclipse.jdt.core.formatter.parentheses_positions_in_try_clause=common_lines
 org.eclipse.jdt.core.formatter.put_empty_statement_on_new_line=true
-org.eclipse.jdt.core.formatter.tabulation.char=tab
+org.eclipse.jdt.core.formatter.tabulation.char=space
```

Comment:
This change is not acceptable and will cause everyone using Eclipse to start having problems with CheckStyle.  Please revert back to tabs here.